### DISCUSSION TITLE
How can I keep the speech recognition constantly active?"

### DISCUSSION THREAD

**Discussion by meanyf** [yasirkula/UnitySpeechToText#9] [Q&A]
The speech recognition depends on the volume of speech, but I need it to listen continuously.

**Comment by yasirkula** [comment]
It's not supported by this plugin. Actually, the native Android backend (SpeechRecognizer) doesn't support it either. And the native iOS backend (SFSpeechRecognizer) AFAIK has a maximum session duration which is an obstacle.
